helix-core/src/main/java/org/apache/helix/task/assigner/ThreadCountBasedTaskAssigner.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Map;
 import java.util.PriorityQueue;
 
+import java.util.Set;
```

Comment:
is this intended part of the change?
As this doesn't look like log level related change